### PR TITLE
TOMEE-2877 Translate to Portuguese: examples/mp-rest-jwt-public-key

### PR DESCRIPTION
**Pull request by Daniel-Dos**
Link to Jira: https://issues.apache.org/jira/browse/TOMEE-2877

### CODE REVIEW

File: examples/mp-rest-jwt-public-key/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+Para ativar o JWT, você precisa anotar sua classe de aplicação REST com a anotação `org.eclipse.microprofile.auth.LoginConfig`.
+Neste exemplo, a classe é `ApplicationConfig`.
+
+Outra coisa que precisa ser feita é configurar a `public key` para verificar a assinatura do JWT anexada no cabeçalho `Authorization'.
+É assinado na criação com a `private key` do issuer.
+Isso é feito para evitar o uso moderado do token enquanto ele viaja do chamador até o endpoint.
+Geralmente, a emissão do JWT ocorre em um módulo ou microsserviço especial responsável pela autenticação dos usuários.
+Neste projeto de amostra, isso acontece no `BookstoreTest`.
+
+Cada tempo de execução de suporte ao MicroProfile JWT deve poder verificar se a assinatura está correta e se o conteúdo assinado não é alterado ao longo do caminho.
```
Comment (willesreis) [inline examples/mp-rest-jwt-public-key/README_pt.adoc:34]:
```suggestion
Cada ambiente de execução que suporta o MicroProfile JWT deveria ser capaz de verificar se a assinatura está correta e se o conteúdo assinado não é alterado ao longo do caminho.
```

File: examples/mp-rest-jwt-public-key/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+Para fazer isso, ele precisa ter acesso a uma `public key`.
+Essa `public key` pode estar no formato `PKCS#8 PEM, JWK ou JWKS`.
+Desde o MP JWT 1.1 (que é suportado pelo TomEE), a chave pode ser fornecida como uma string na propriedade de configuração `mp.jwt.verify.publickey` ou como um local ou URL de arquivo especificado em `mp.jwt.verify.publickey.location` na propriedade de configuração.
+
+Neste projeto de amostra, você pode ver a primeira opção.
```
Comment (willesreis) [inline examples/mp-rest-jwt-public-key/README_pt.adoc:39]:
```suggestion
Neste projeto de exemplo, você pode ver a primeira opção.
```

File: examples/mp-rest-jwt-public-key/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+O arquivo `src/main/resource/META-INF/microprofile-config.properties` contém a seguinte entrada:
+
+[source,properties]
+----
+mp.jwt.verify.publickey=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAlivFI8qB4D0y2jy0CfEqFyy46R0o7S8TKpsx5xbHKoU1VWg6QkQm+ntyIv1p4kE1sPEQO73+HY8+Bzs75XwRTYL1BmR1w8J5hmjVWjc6R2BTBGAYRPFRhor3kpM6ni2SPmNNhurEAHw7TaqszP5eUF/F9+KEBWkwVta+PZ37bwqSE4sCb1soZFrVz/UT/LF4tYpuVYt3YbqToZ3pZOZ9AX2o1GCG3xwOjkc4x0W7ezbQZdC9iftPxVHR8irOijJRRjcPDtA6vPKpzLl6CyYnsIYPd99ltwxTHjr3npfv/3Lw50bAkbT4HeLFxTx4flEoZLKO/g0bAoV2uqBhkA9xnQIDAQAB
+----
+
+== trabalhando com JWT
```
Comment (willesreis) [inline examples/mp-rest-jwt-public-key/README_pt.adoc:47]:
```suggestion
== Trabalhando com JWT
```

File: examples/mp-rest-jwt-public-key/README_pt.adoc [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+A classe `BookResource` neste projeto de amostra mostra dois casos em que você pode usar a especificação MP JWT: obter o valor de uma declaração JWT e o controle de acesso baseado em função dos endpoints REST.
```
Comment (willesreis) [inline examples/mp-rest-jwt-public-key/README_pt.adoc:49]:
```suggestion
A classe `BookResource` neste projeto de exemplo mostra dois casos em que você pode usar a especificação MP JWT: obter o valor de uma declaração JWT e o controle de acesso baseado em função dos endpoints REST.
```

File: examples/mp-rest-jwt-public-key/README_pt.adoc [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+=== Obtenção de valores de claim
+
+O token da web JSON (JWT) anexado no cabeçalho HTTP `Authorization` é essencialmente um objeto JSON que contém vários atributos.
```
Comment (willesreis) [inline examples/mp-rest-jwt-public-key/README_pt.adoc:53]:
```suggestion
O JSON Web Token (JWT) anexado no cabeçalho HTTP `Authorization` é essencialmente um objeto JSON que contém vários atributos.
```

File: examples/mp-rest-jwt-public-key/README_pt.adoc [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+    @POST
+    @RolesAllowed("manager")
+    public void addBook(Book newBook) {
+        booksBean.addBook(newBook);
+    }
+----
+
+== O teste do bookstore
+
+O projeto de amostra contém um teste Arquillian (`org.superbiz.bookstore.BookstoreTest`) usado por alguns motivos:
```
Comment (willesreis) [inline examples/mp-rest-jwt-public-key/README_pt.adoc:103]:
```suggestion
O projeto de exemplo contém um teste do Arquillian (`org.superbiz.bookstore.BookstoreTest`) usado por alguns motivos:
```

File: examples/mp-rest-jwt-public-key/README_pt.adoc [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+
+* Generating the JSON web token (JWT)
```
Comment (willesreis) [inline examples/mp-rest-jwt-public-key/README_pt.adoc:105]:
```suggestion
* Geração do JSON Web Token (JWT)
```

File: examples/mp-rest-jwt-public-key/README_pt.adoc [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+* Apresentando o comportamento do TomEE em diferentes situações
```
Comment (willesreis) [inline examples/mp-rest-jwt-public-key/README_pt.adoc:106]:
```suggestion
* Apresentação do comportamento do TomEE em diferentes situações
```

**Comment by Daniel-Dos** [comment]
thanks @willesreis  for review .

**Comment by Daniel-Dos** [comment]
thanks.

**Pull request merged**